src/components/Book.js

```diff
@@ -15,28 +18,44 @@ const Book = () => {
 
   const booksArray = Object.values(books)[0];
   return (
-    <tbody className="table_body">
+    <>
       {booksArray.map((book) => (
-        <tr key={book.id}>
-          <td>
-            {book.title}
-            {' '}
-            by
-            {' '}
-            {book.author}
-          </td>
-          <td>
-            <button
-              type="button"
-              className="remove_btn"
-              onClick={() => handleRemoveBook(book.id)}
-            >
-              Remove
-            </button>
-          </td>
-        </tr>
+        <section className="each-book" key={book.id}>
+          <div className="bk-details">
+            <h5 className="entry-category">Category</h5>
+            <h3 className="entry-title">{book.title}</h3>
+            <h5 className="entry-author">{book.author}</h5>
+            <div className="bk-actions">
+              <button type="button" className="btn-type1">Comments</button>
+              <hr className="vertical line-1" />
+              <button
+                type="button"
+                className="remove_btn btn-type1"
+                onClick={() => handleRemoveBook(book.id)}
+              >
+                Remove
+              </button>
+              <hr className="vertical line-1" />
+              <button type="button" className="btn-type1">Edit</button>
+            </div>
+          </div>
+          <div className="progress-cont">
+            <CircularProgressbar value={66} />
+          </div>
+          <div>
+            <span className="percentage">64%</span>
+            <br />
+            <span className="entry-completed">Completed</span>
+          </div>
+          <hr className="vertical line-2" />
+          <div>
+            <h4 className="entry-chapter">CURRENT CHAPTER</h4>
+            <h4 className="chapt-no">Chapter 17</h4>
+            <button type="button" className="btn-type2">UPDATE PROGRESS</button>
+          </div>
+        </section>
```

Comment:
- [ ] Please ensure that this section of the book display has enough space to the right when compared to the original design.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/222900032-a47173d2-6b60-4ad7-b4ee-3cca445571a2.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/222900081-2dd3e106-2da0-4acb-8d6f-4d047fa2ba3f.png)